### DISCUSSION TITLE
When saving screenshots, some settings don't take effect

### DISCUSSION THREAD

**Discussion by jromazanov** [pyvista/pyvista#4081] [Q&A]
EDIT: Sorry, I realized this belongs rather into the pyvistaqt discussion section. I opened a [discussion there.](https://github.com/pyvista/pyvistaqt/discussions/325)

I'd like to make a 3D mesh plot, define a camera rotation in the script, show an interactive window with some adjusted settings (e.g. camera rotation, canvas background color), and finally save a screenshot into `test.png`.

This works fine with pyvista's `Plotter` class, but apparently not with pyvistaqt's `BackgroundPlotter` class.
Below is a minimal example. If I run with `plotType = 'pv'`, it works fine, the screenshot has the same rotation and background color as the interactive window. But if I run with `plotType = 'pvqt'`, the screenshot forgets the camera rotation and always uses the same default rotation. 

Question: Is there a different way I need to pass the settings and save the screenshot? Or is this a bug?

(Interestingly, other settings are transferred correctly to the pyvistaqt screenshot, e.g. `pl.view_yz()` would take effect in both interactive window and screenshot.)

```
import pyvista as pv
import pyvistaqt as pvqt

# uncomment one:
#plotType = 'pv'
plotType = 'pvqt'

# create plotter
if plotType == 'pv':
    pl = pv.Plotter()
else:
    pl = pvqt.BackgroundPlotter()

# do plot
cube = pv.Cube()
pl.add_mesh(cube, show_edges=True)

# change camera angle
pl.camera.azimuth = 45.0

# change background color
pl.set_background('white')

# show figure and take screenshot
if plotType == 'pv':
    pl.show(screenshot='test.png')
else:
    screenshot = pl.screenshot('test.png')
    pl.app.exec_()
```

**Comment by adeak** [comment]
Locking this as this is indeed qt-specific and there's a duplicate discussion. Please see [that](https://github.com/pyvista/pyvistaqt/discussions/325).
